endpoints.json

```diff
@@ -25,5 +25,8 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves an article object with the correct properties when given a valid article_id number"
```

Comment:
Is there any other info we could give the user about this end point? eg an example response?